### PR TITLE
User picker logic and stuff

### PR DESCRIPTION
**Pull request by bojakowsky**
Stuff, user stuff, it's the "starter" user picker. It can be easily extended for our needs in future.

### CODE REVIEW

File: TaskVlopperTests/Controllers/AccountControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace TaskVlopper.Controllers.Tests
+{
+    [TestClass()]
+    public class AccountControllerTests
+    {
+        [TestMethod()]
+        public void UsersTest()
+        {
+            Assert.Fail();
```
Comment (mszynka) [inline TaskVlopperTests/Controllers/AccountControllerTests.cs]: If tests fails then no merging PR, right? :wink:

File: TaskVlopperTests/TaskVlopperTests.csproj [hunk 2]
```diff
@@ -0,0 +1,90 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (mszynka) [inline TaskVlopperTests/TaskVlopperTests.csproj]: Why, O, why do we need another test project in solution? I simply don't get that. Error or intended?

**Comment by bojakowsky** [comment]
It was created by mistake ;(

**Comment by mszynka** [comment]
Your bad anyway :laughing:

**Pull request merged**